backend/entityservice/logger_setup.py

```diff
@@ -0,0 +1,90 @@
+import os
+import logging.config
+import structlog
+
+import yaml
+
+
+def setup_logging(
+    default_path='logging.yaml',
+    default_level='INFO',
+    env_key='LOG_CFG'
+):
+    """
+    Setup logging configuration
+    """
+    path = os.path.dirname(os.path.abspath(__file__)) + '/' + default_path
```

Comment:
how about `os.path.join` instead?

Reply:
I'll raise you a [pathlib](https://docs.python.org/3/library/pathlib.html#module-pathlib)